src/Web/Handler.hs

```diff
@@ -147,6 +200,161 @@ getTheory idx = do
     liftIO $ withMVar (theoryVar yesod) $ return. M.lookup idx
 
 
+getLemmaPlaintext :: Int -> TheoryPath -> Handler String
+getLemmaPlaintext nr path = do
+    let lname = case path of 
+            (TheoryEdit n) -> n 
+            _ -> "this should never be visible"
+    eitherTheory <- getTheory nr
+    let lemmaItem = case eitherTheory of 
+            (Just (Trace thy)) -> lookupLemma lname (tiTheory thy)
+            _ -> Nothing
+    let plaintext =  fromMaybe "Enter your new Lemma" $ get lPlaintext <$> lemmaItem
+    return plaintext
+
+
+getStartingProof ::  System -> IncrementalProof
+getStartingProof gsys = LNode (ProofStep (Sorry Nothing) (Just gsys)) M.empty
```

Comment:
I think you can delete this function and use `unproven (Just gsys)` instead (from Theory.Proof).